cp23/hpi-fun2/cubes/problem.tex

```diff
@@ -0,0 +1,32 @@
+\input{template.tex}
+
+\begin{document}
+
+\makeheader
+
+``Setz dich nicht so nah vor den Fernseher - du bekommst noch viereckige Augen!''
+``Don't sit so close to the TV - you'll get square eyes!''
```

Comment:
(You may want to take a look at the `epigraph` package)
![Screenshot from 2023-07-08 14-04-16](https://github.com/compprog-lecture-tools/fun-contest-2023/assets/33495148/6c2dfceb-0c9f-444f-a242-4280f267ddc4)